core/src/trezor/messages.py

```
@@ -17,6 +17,7 @@
 if TYPE_CHECKING:
     from typing import TypeGuard
     from trezor.enums import AmountUnit  # noqa: F401
+    from trezor.enums import AptosTransactionType  # noqa: F401
```

Ruff (0.11.9)

Unused `noqa` directive (unused: `F401`)

Remove unused `noqa` directive

(RUF100)